pilots/10_things_to_know_about_pilots.Rmd

```diff
@@ -0,0 +1,189 @@
+
+There are a few ways to explore these unknowns in your pilot study. You might wish to include additional exploratory outcome questions, collect extra covariate data, or use a variety of treatments drawing on innovative or untested ideas. Another great way to identify unknowns is through open-ended questions of study participants. Regardless of your study design, you might consider surveying or interviewing participants to ask "what they think about topic X," or "what comes to mind when they hear the term Y." The world is often more complex than researchers model in their study designs, and study participants often have more diverse perspectives and relationships with the issues at hand than researchers expect. 
+
+Through this process, you may identify novel research questions, potential theoretical dimensions or causal mechanisms, and hypotheses that you had not originally formulated when you began studying the topic. Exploring unknowns can therefore lead to refining the ideas you had originally conceived of, as you are unlikely to have determined the best version of your study from the beginning. It can also lead to entirely novel considerations and ideas altogether, some of which you may be able to incorporate into the current study or future studies.
+
+Overall, pilots afford a wonderful opportunity for open-ended exploration and idea generation that should not be overlooked.
+
+# 10. A pilot is part of a broader sequence of research activities
+
+While we've talked about research in terms of "pilots" and "main studies," this is an oversimplification of the research process. Indeed one can (and often should) employ multiple pilot studies, perhaps with different sample sizes, to evaluate different questions and to continue refining one's research design as new questions and possibilities emerge. Pilot studies certainly do set the stage for main studies as well as subsequent "follow-up" studies, but there may not be a sharp demarcation between these kinds of studies. Pilot studies contain findings in their own right and can certainly be shared, presented, and published. As depicted in the graphic, pilot studies do not merely lead to main studies. They can also help one develop new ideas and they may serve as a venue through which one can contribute to or coordinate with the broader research community around topics of shared interest. Indeed, there may be multiple research teams working on similar topics simultaneously.
+
+
+<center>
+![The Role of Pilots in Experimental Research](research_sequence.png){width=50%} 

```

Comment:
Yep. Just send it and we'll distribute it in the https://github.com/egap/methods-guides directory (we'll make a directory called "pilot_studies" or something).